File: ServicioBarberia/src/main/java/com/ceiba/barberia/dominio/entidades/Novedad.java
package com.ceiba.barberia.dominio.entidades;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Novedad {
	
	private Long id;
	private Date fechaInicio;
	private Date fechaFin;
	private Barbero barbero;
	private String descripcion;
	private Boolean festivo;
}

File: FrontBarberia/src/app/core/model/novedad.ts
import { Barbero } from './barbero';

export class Novedad {

    constructor(
        public id: number,
        public fechaInicio: Date,
        public fechaFin: Date,
        public barbero: Barbero,
        public descripcion: string,
        public festivo: boolean
    ) { 
        if(barbero == null) {
            this.barbero = new Barbero(null, "");
        }
    }
}